### PR TITLE
Switch spinning demo to evil ad demo

### PR DESCRIPTION
**Pull request by ConnorGBrewster**
Switching because spinning has issues (servo/servo#11970) and evil ad is much cooler.

---
This change is [Reviewable](https://reviewable.io/reviews/browserhtml/browserhtml/1139)

### CODE REVIEW

File: src/about/newtab/tiles.json [hunk 1]
```diff
@@ -70,9 +70,9 @@
     , "src": "./tiles/demo-transparent_rects.png"
     }
   , "11":
-    { "title": "Demo: Spinning"
-    , "uri": "./demos/spinning.html"
-    , "src": "./tiles/demo-spinning.png"
+    { "title": "Demo: Evil Ad"
+    , "uri": "./demos/evil_ad/physics/with-ad.html"
```
Comment (Gozala) [inline src/about/newtab/tiles.json]: This points to a file that does not exists. Can you please fix and resubmit ?
I had to revert this change.

Comment (ConnorGBrewster) [inline src/about/newtab/tiles.json]: that odd, somehow this was working on my local machine 😕
